content/docs/06-integrations/00-byoos.md

```diff
@@ -0,0 +1,239 @@
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+
+# Bring Your Own Operating System (BYOOS) 
+
+The [Bring Your Own Operating System (BYOOS)](/https://docs.spectrocloud.com/cluster-profiles/byoos) enables the ability for you to use a custom Operating System (OS) with Palette. Palette comes with several OS out-of-the-box, but for some users, the existing list of OS may not meet their needs. 
+
+Using your custom OS provides several benefits, including the ability to control your own dependencies, improve performance, and ensure compatibility with your existing applications. With BYOOS, you can choose the OS that best fits your needs, whether it's a commercial or open-source distribution, and integrate it with your Kubernetes clusters. The BYOOS pack can be used with both Edge and non-Edge environments. 
+
+# Versions Supported
+
+**1.0.x**
+
+<br />
+
+<Tabs>
+
+<Tabs.TabPane tab="Edge" key="edge">
+
+## Prerequisites 
+
+To use the Edge BYOOS pack, you must have the following:
+
+<br />
+
+- The Edge Provider images are created and uploaded to a container registry. Refer to the [Bring Your Own OS](/clusters/edge/edgeforge-workflow/build-kairos-os) guide to learn how to create a custom OS. Refer to the [Build Images](/clusters/edge/edgeforge-workflow/build-images) guide for steps on how to create the Edge artifacts and how to upload your custom OS to a registry.
+
+
+- Palette 3.3.0 or greater.
+
+## Parameters
+
+The BYOS Edge OS pack supports the following parameters. 
+
+### Parameters
+
+| Parameter            | Description                                            | Type |
+|----------------------|--------------------------------------------------------|------|
+| `pack:content:` | Specifies the content of the **BYOS Edge OS** pack. | map |
+| `pack.content.images` | Specifies a list of OS images to use with the pack. | list |
+| `pack.content.images.image` | An OS image to use with the pack. | string|
+| `system.uri` | The system URI specifies the BYOOS image's location. | string|  
+
+
+
+  ```yaml
+  pack:
+  content:
+    images: 
+      - image: '{{.spectro.pack.edge-native-byoi.options.system.uri}}'
+      # - image: example.io/my-other-images/example:v1.0.0 
+      # - image: example.io/my-super-other-images/example:v1.0.0 
+    
+  options: 
+    system.uri: example.io/my-images/example-custom-os:v1.4.5
+  ```
+
+## Usage
+
+BYOOS enables you to use a custom OS for your Edge host. You can use this feature to customize the desired specifications of your OS layer in the Edge host. You can reference the custom OS through the BYOOS pack.
+
+
+To use a custom OS, you must include all the Edge artifacts and provider images required by the Edge Installer in the custom OS. Refer to the [Bring Your Own OS](/clusters/edge/edgeforge-workflow/build-kairos-os) guide for steps on how to create a custom OS that includes all the required components for the Edge Installer.
```

Comment:
In the first sentence, is "provider" correct, or is "provide" intended?

Reply:
The term is "Provider Images."

Reply:
It's just provider